py4DSTEM/process/diskdetection/diskdetection.py

```diff
@@ -404,6 +446,13 @@ def find_Bragg_disks(datacube, probe,
                                                         DFT upsampling
         upsample_factor      (int) upsampling factor for subpixel fitting (only used when subpixel='multicorr')
         verbose              (bool) if True, prints completion updates for serial execution
+        filter_function      (callable: lambda or function of one argument) filtering function to apply
+                             to each diffraction pattern before peakfinding. must be a function of only
+                             one argument (the diffraction pattern) and able to be pickled by the dill library.
+                             This is useful for doing noise reduction or binning on the fly for memory mapped
+                             datasets or for parallel operation. The shape of the returned DP must match the shape
+                             of the probe kernel. When using distributed mode, the function must contain
+                             all its own imports.
```

Comment:
the fact that it was necessary to add this 17 times is evidence of poor code architecture.  My bad :/